ci/docker/runtime_functions.sh

```diff
@@ -1227,11 +1227,18 @@ nightly_tutorial_test_ubuntu_python2_gpu() {
 nightly_java_demo_test_cpu() {
     set -ex
     cd /work/mxnet/scala-package/mxnet-demo/java-demo
-    make javademo
+    make java_ci_demo
     ./bin/java_sample.sh
     ./bin/run_od.sh
 }
 
+nightly_scala_demo_test_cpu() {
+    set -ex
+    cd /work/mxnet/scala-package/mxnet-demo/scala-demo
+    make scala_ci_demo
+    bash bin/demo.sh
```

Comment:
Sync up script filenames and calling method (bash x.sh vs ./x.sh) as well

Reply:
Syncing up the script filenames would be a bigger change, as we will have to clean up in places where the demo tutorials is being referred. eg : README, Website etc. I don't want to bring in any other side effects here

I can change the calling method to bash though in the nightly runs.